client/src/models/SignUpUser.ts

```diff
@@ -15,7 +15,7 @@ interface SignUpUser {
     [SignUpFields.MAIL]?: string,
     [SignUpFields.COUNTY]?: number,
     [SignUpFields.SOURCE_ORGANIZATION]?: string,
-    [SignUpFields.LANGUAGES]: Language[]
+    [SignUpFields.LANGUAGES]?: Language[]
```

Comment:
Why optional?

Reply:
Because I want to initilize user information empty at first

Reply:
so you can just init it with empty array instead

Reply:
Yes but why?
I can just init the entire object empty